### PR TITLE
Bobo

### PR DESCRIPTION
**Pull request by BOBO-LU**
finsih color detect model

### CODE REVIEW

**Review by schuaBob — COMMENTED** [review]
need some change

File: backend/ontology/onto.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+file_path = "file: // ./ontology/owl/"
+
 
 def get_location(location_tag):
     rtr = []
     province_list = ['市', '縣', '區']
     try:
         onto_location = get_ontology(
-            "file://./ontology/location_onto.owl").load()
+            file_path + "location.owl").load()  # 還沒處理file path variable
```
Comment (schuaBob) [inline backend/ontology/onto.py:11]: path problem, need fix

File: backend/ontology/onto.py [hunk 2]
```diff
@@ -41,7 +43,9 @@ def locate_area(location_tag):
     except Exception as e:
         print(e)
         return []
+
     print('ontology location:', rtr)
```
Comment (schuaBob) [inline backend/ontology/onto.py:47]: remove this

File: frontend/.gitignore [hunk 3]
```diff
@@ -61,4 +61,6 @@ buck-out/
 
 dev.js
 google-services.json
-*.json
\ No newline at end of file
+
+*.json
```
Comment (schuaBob) [inline frontend/.gitignore:65]: 幫我改成env.json

**Review by BOBO-LU — COMMENTED** [review]
finish file path modify

**Review by schuaBob — APPROVED** [review]

**Pull request merged**